README.md

```diff
@@ -85,6 +85,6 @@ Want to chat with us, discuss changes, or just hang out? We have a Discord serve
 
 This mod was not created for any harmful purposes, such as harming Sketchware; quite the opposite, actually. It was made to keep Sketchware alive by the community for the community. Please use it at your own discretion and consider becoming a Patreon backer to support the developers. Unfortunately, other ways to support them are not working anymore, so Patreon is the only available option currently. You can find their Patreon page [here](https://www.patreon.com/sketchware).
 
-We do NOT permit the modification of this repository then post it on Play Store or any other app store as a new app. Keep in mind that this project is still a mod. Competing with the original company using a mod is considered illegal, and we strongly discourage such behavior.
+We do NOT permit publishing Sketchware Pro on Play Store or on any other app store. Keep in mind that this project is still a mod. Unauthorised modding of apps is considered illegal and we discourage such behavior.
```

Comment:
> We do NOT permit publishing Sketchware Pro...
So by this change, you mean anyone can change the app name to something else and publish it as long as it's not named Sketchware Pro?

Reply:
check now

Reply:
Looks good. Also, why `unauthorised` and not `unauthorized`? UK moment.

Reply:
the readme uses US English so stick to that please

Reply:
fixed